client/app/components/calendar/filters/RoomFilters.jsx

```diff
@@ -7,22 +7,25 @@ import './filters.scss';
 export default class RoomFilters extends React.Component {
   static propTypes = {
     conferenceRooms: PropTypes.arrayOf(ConferenceRoomSchema.only('id')).isRequired,
-    onEnabled:       PropTypes.func.isRequired,
-    onDisabled:      PropTypes.func.isRequired,
-    filters:         PropTypes.arrayOf(PropTypes.number)
+    onEnabled: PropTypes.func.isRequired,
+    onDisabled: PropTypes.func.isRequired,
+    filters: PropTypes.arrayOf(PropTypes.number)
   };
 
   static defaultProps = {
     filters: []
   };
 
   render() {
-    let filters = this.props.conferenceRooms.map(conferenceRoom => (
+    let filters = this.props.conferenceRooms.sort(this._roomCompare).map(conferenceRoom => (
       <Filter conferenceRoom={conferenceRoom}
```

Comment:
I think that we don't need any more to pass `conferenceRoom`. Good job - #LOD :D